scripts/restore

```diff
@@ -58,6 +58,7 @@ fi
 #=================================================
 
 ynh_restore_file --origin_path="/etc/nginx/conf.d/$domain.d/$app.conf"
+ynh_restore_file --origin_path="/etc/nginx/conf.d/$domain.d/$app.d/"
```

Comment:
This causes a warning due to these lines in the helpers:

https://github.com/YunoHost/yunohost/blob/394907ff0df218ec926f43c2e2ba006c68cc3f8a/helpers/backup#L292-L295